hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -419,6 +452,9 @@ <h4 class="modal-title" id="save-collection-warning-title">Deleted Resources</h4
                 removed_row_from_colletion.remove().draw(false);
             }
             $('#save-new').show();
+
+            // Rebind click event for new row
+            $("#collection-table-candidate td").click(onCollectionTableRowClick);
```

Comment:
These are small style changes I did while working on this branch, so I didn't think they needed a new PR. We should have this deployed somewhere so they can be tested.